models/BaseConfig.cfc

```diff
@@ -993,17 +993,17 @@ component accessors="true" {
 
 	/**
 	* Add a single Gateway instance to the config
+	* @gatewayId An event gateway ID to identify the specific event gateway instance.
+	* @type The event gateway type, which you select from the available event gateway types, such as SMS or Socket.
 	* @cfcPaths The absolute path to the listener CFC or CFCs that handles incoming messages.
+	* @mode The event gateway start-up status; one of the following: automatic, manual, disabled
 	* @configurationPath A configuration file, if necessary for this event gateway type or instance.
-	* @gateway Id An event gateway ID to identify the specific event gateway instance.
-	* @mode The event gateway start-up status; one of the following: Automatic, Manual, Disabled
-	* @type The event gateway type, which you select from the available event gateway types, such as SMS or Socket.
 	*/
-	function addGatewayInstance(array cfcPaths,
```

Comment:
Just curious why these were rearranged?

Reply:
To have the same order as in the coldfusion administrator.